backend/backend/settings/common.py

```diff
@@ -224,6 +224,18 @@ def to_bool(value):
 HTTP_CLIENT_TIMEOUT_SECONDS = int(os.environ.get('HTTP_CLIENT_TIMEOUT_SECONDS', 30))
 
 LOGGING_USE_COLORS = to_bool(os.environ.get('LOGGING_USE_COLORS', True))
+
+
+# Without this import statement, hfc.fabric outputs DEBUG-level logs regardless of LOGGING configuration.
+# Importing the module early ensures these steps happen in order:
+# 1. hfc.fabric is imported and all the associated loggers are configured  (log level, handler, etc...)
+# 2. Loggers are re-configured with the `LOGGING` statement, overriding the configuration from step 1
+try:
+    import hfc.fabric
+except:
```

Comment:
What exception you can have here ? You should catch the right one :)

Reply:
Fixed: https://github.com/SubstraFoundation/substra-backend/pull/338/commits/c8600c1e9b8d95f64ab4ec25fc06fec7ca596e13

This try/catch was intended to show that not importing fabric is a valid use case. 

However, if we want to conditionally stop using fabric in the future, we probably will want to still do the `import`.